.travis-ci.sh

```diff
@@ -11,6 +11,19 @@ COVERITY_SCAN_BUILD_URL="https://scan.coverity.com/scripts/travisci_build_coveri
 
 PYCHECKER_BLACKLIST="threading,unittest,cmd,optparse,google,google.protobuf,ssl,fftpack,lapack_lite,mtrand"
 
+LINT_BLACKLIST=$(cat <<EOLBL
+        -wholename "./common/protocol/Ola.pb.*" -or \
+        -wholename "./common/rpc/Rpc.pb.*" -or \
+        -wholename "./common/rpc/TestService.pb.*" -or \
+        -wholename "./common/rdm/Pids.pb.*" -or \
+        -wholename "./config.h" -or \
+        -wholename "./plugins/*/messages/*ConfigMessages.pb.*" -or \
+        -wholename "./tools/ola_trigger/config.tab.*" -or \
+        -wholename "./tools/ola_trigger/lex.yy.cpp" -or \
+        -wholename "./plugins/ftdidmx/FtdiDmxPluginDescription.h"
```

Comment:
BTW why drop it? Even now it is auto-generated and there is really no reason to check it, just not yet with the contents that annoys the lint checks....

Reply:
There's a bit of me thinks we should hopefully be able to fix it failing the lint and that generally files we self auto-generate we should be able to make lint clean (compared to say the output of the lexer where we don't have any control of that).

Plus if we're going to go down that route, we should ignore ALL the plugin descriptions, not just one, then this needs implementing too really https://github.com/OpenLightingProject/ola/issues/691 (otherwise random developers will find lint-dirty code). I'd also want to double-check what we ship in terms of tarballs, but I think this is built locally so that's not relevant.